packages/odyssey-storybook/src/components/odyssey-mui/Tag/Tag.mdx

```diff
@@ -0,0 +1,49 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+
+<Meta anchor />
+
+# Tag
+
+Use tags to help describe and differentiate an entity or object. Think of them as “adjectives” in your UI toolbox that make navigating and parsing content easier.
+
+TagList UI is simple. It consists of typography and spacing within a neutral container. Color, weight, and spacing help distinguish it from Button.
```

Comment:
```suggestion
TagList UI is simple. It consists of typography and spacing within a neutral container. Color, weight, and spacing help distinguish it from `Button`.
```

Reply:
@gretchen-okta is working on some official guidelines for when to use "button," "Button," and "`<Button>`". AFAIK the tentative content rule is: 

- use "button" when talking about general UI, "Clicking a button"
- use "Button" when talking about a component or pattern generally, "Use Button for in-page interactions"
- use "`<Button>`" when talking about React component APIs, "`Button` takes the `variant` prop"

We should make this official somewhere, and I'm sure Gretchen would be interested in getting eng feedback there as well.

Reply:
@edburyenegren-okta @KevinGhadyani-Okta Sorry for the week-long delay! I let this float by. When talking about a component I'd prefer to follow [Material](https://m3.material.io/components/all-buttons), [Shopify](https://polaris.shopify.com/components/button), etc and not make it a proper noun when talking about a component. 

- use "button" when talking about general UI, "Clicking a button"
- use "button" when talking about a component or pattern generally, "Use button for in-page interactions"
- use "<Button>" when talking about React component APIs, "Button takes the variant prop"